packages/zosfiles/CHANGELOG.md

```diff
@@ -2,14 +2,18 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Allows for passing a `.zosattributues` file path for the download encoding format via the `attributes` option on the `Download.ussFile` method. [#2322](https://github.com/zowe/zowe-cli/issues/2322)
```

Comment:
Ideally (and if it makes sense), this should match the other changelog.

Reply:
I know it is slightly different, but I believe it's more accurate for them to be slightly different.

There is no concept of a `--attributes` or a `zowe files download` command as far as the `packages/zosfiles` is aware. 😅